### ISSUE TITLE
🌅 Visors Project — The start of the wind down  

### ISSUE THREAD

**Issue by afroleft** [DoESLiverpool/covid19#105]
Team -

10,000 visors shipped. Something that we should all be proud of.  

@jackreason85 has been doing a neat job of managing the orders, fulfilment tracking and deliveries. I don’t think we dropped the ball on one order. So real thank yous to @DoESsean, Irene, @amcewen and Lyn who helped him there. 

I should have fed more of the messages back at the time, but we’ve had everything from front line teams working ICU bursting in to tears when we made them a shipment on the same day they requested, late night / early hours WhatsApp chats with relieved and genuinely grateful procurement teams, even executive leadership teams from CCGs and healthcare trusts prompting their corporate comms teams to publish stories thanking DoES Liverpool for what we have done. 

## What is the goal and how do we keep score?
Over 40 volunteers rolled their sleeves up to help this project along. We will all have slightly different views of what this project was for, but for me and most of the people I’ve been working with, it’s been about propping-up the critical need healthcare workers with protective visors until the supply chains from the Far East came back online. 

We know visors offer essential splash protection from covid patients. 

If we’ve kept 10 or more frontline workers healthy, well and fully able to do their job of caring for patients or cleaning the hospitals, then this project has been more than a success.   

## What’s next? 
With our partners we have worked through nearly 70 orders. The backlog is close to empty.

We have 2 rolling orders with Liverpool Heart & Chest and East Lancs that are likely going to wind down over the next few weeks. 

I’ve taken calls with a number of procurements leads and front line teams and news is; supplies are now up and they are in a good position for protective visors.   

With that, we’ll look to wind-up the visor production over the next few weeks. 

We’ll follow @amcewen and @JackiePease lead since we might want to produce a stock pile, just incase we run in to a critical need order. 
## One final push 
We’ll switch gears on the comms side to do another reach of final calls to prompt those workers in need to reach out for a box of visors.

I’ll follow up in another thread //cc @hannahforbes but the campaign will look something like this: 

* We’ve produced 10,000 visors for hospitals, care homes, GP practices, pharmacies, physios and more
* This is your last opportunity to request visors *while materials last*
* We’re might even open up to other workers that could benefit from protective visors; e.g. bus drivers, security guards, retail workers, pharmacies etc etc
  
After this, we’ll wind our neck back in, updating our little website to show the landmarks and progress we made over the last 5 weeks.

## Other ways we can help 
I’m linking Adrian with a number of the healthcare teams to see if there are other ways we might be able to help. 

If there’s something that looks like a fit, I’m sure he’ll be following up with us all. 

Stay well folks. This project has been way more inspiring than it has been exhausting. And it has been exhausting! 

Look forward to joining you all for a beer some time soon,

There is one more thing —> [Link](https://www.youtube.com/watch?v=Ee3Jh5qRDMo)

**Comment by JackiePease** [comment]
Other workers who might benefit: posties, warehouse staff

**Comment by afroleft** [comment]
@JackiePease Good call. Is Duncan good to start reaching out to these organisations?

**Comment by JackiePease** [comment]
He's phoning some people tomorrow. He's in on Tuesday, so I'll check then.On 3 May 2020 17:30, Tom Darlow <notifications@github.com> wrote:
@JackiePease Good call. Is Duncan good to start reaching out to these organisations?

**Comment by afroleft** [comment]
## Feedback from Dave MacMillan at Liverpool Heart & Chest.

Following our conversation I said I would put into writing the feedback I gave you.

Firstly, thank you to you and your team for the amazing support you have provided not only LHCH but our partner Trusts across Cheshire and Merseyside. It has been truly amazing to see people pull together to support the efforts. Every time I walk into our critical care areas I see our nurses and medics wearing the visors your team have created. Speaking to the nurses they are appreciative of the support which ensures they are kept safe.

Secondly, I initially trialled these visors with our surgeons who have provided some great feedback. It appears these are the preferred visors for our surgeons who use loupes. Once again, whenever I go into theatre and our cath labs I see the clinical teams wearing these visors.

Thirdly, your visors are the ones which our community team request regularly and they have been back in touch this morning to request more of the visors. They are the preferred type when they are out in the communit.

Finally, the product itself has proven to be reusable and survives a number of high temperature washes through our Theatre washers which has ensured our stock levels remain sufficient to support other Trusts across the city.

Once again, thank you and your team for what has been a sterling performance and thank you for joining us in our efforts to support our staff and patients during the pandemic.

**Comment by afroleft** [comment]
![image](https://user-images.githubusercontent.com/1761311/81733918-a30a9980-948a-11ea-814e-dff9cc37ba0c.jpeg)
